v5/api/c/imu.rst

```diff
@@ -5,6 +5,8 @@
 VEX Inertial Sensor C API
 =========================
 
+.. note:: ``pros::IMU`` can also be used to refer ``pros::Imu``
```

Comment:
```suggestion
```
Remove this, the new alias is only relevant to the C++ API. (delete line 9 as well)